where-for-dinner/accelerator.axl

```diff
@@ -0,0 +1,12 @@
+engine {
+
+  {
+    Include({'**'})
```

Comment:
That's... quite a change from the existing version. I assume this is what we want / can do atm?

Reply:
Yes, we have to add instructions for deploying on Tanzu Platform when we get an environment for testing them